website/scripts/should-deploy-docs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+local NODE_OUPUT=$(node ./scripts/should-deploy-docs.js)
```

Comment:
I've tried, but working in bash is a nightmare. So the main logic is hidden in node 😅

Reply:
Now that I think about it, maybe this bash script is redundant. I've added it to export `SHOULD_DEPLOY_DOCS` variable to pipeline, but it's only used to terminate the pipeline (which is being done here and could be moved to JS file as well).